apps/condo/domains/marketplace/schema/Invoice.js

```diff
@@ -83,31 +85,35 @@ const sendPush = async ({ originalInput, userId, propertyId, unitName, unitType,
                                 userId,
                                 residentId: resident.id,
                                 invoiceId: updatedItem.id,
-                                url: `${conf.SERVER_URL}/invoice/${updatedItem.id}`,
+                                url: `${conf.SERVER_URL}/payments`,
                             },
                         },
                     })
                 } else {
                     const ticket = await getById('Ticket', updatedItem.ticket)
-                    await sendMessage(context, {
-                        dv: 1,
-                        sender: { dv: 1, fingerprint: 'invoice_afterChange' },
-                        to: { user: { id: userId } },
-                        type: MARKETPLACE_INVOICE_WITH_TICKET_PUBLISHED_MESSAGE_TYPE,
-                        uniqKey,
-                        meta: {
+                    const ticketSource = await getById('TicketSource', ticket.source)
+                    if (ticketSource.type !== TICKET_SOURCE_TYPES.MOBILE_APP_RESIDENT ) {
```

Comment:
As far as I know, resident mobile app use `MOBILE_APP` source type, not `MOBILE_APP_RESIDENT`